Doc/library/pathlib.rst

```diff
@@ -1289,6 +1289,29 @@ Reading directories
    raised.
 
 
+.. attribute:: Path.dir_entry
+
+   In a path yielded from :meth:`Path.iterdir`, this attribute stores a
```

Comment:
```suggestion
   In a path yielded from :meth:`Path.iterdir`, this attribute stores an
```

I would use "an" since what will be rendered is "an os.DirEntry" and when I read it, I also read the "os" part.